app/scripts/models/WorkspaceResolver.js

```diff
@@ -281,9 +283,9 @@ define(['backbone', 'FLOOD'],
         x.trigger('requestRender');
 
         // update runner
-        x.trigger('updateRunner');
+        x.trigger('update-node');
```

Comment:
node and workspace has `updateRunner` events but they are different and sometimes node needs to trigger workspace's event. so I renamed the node's event to allow a node trigger workspace's event